_install_public.mdx

```diff
@@ -1,7 +1,7 @@
-In order to use our service you need to install the [pganalyze collector](https://github.com/pganalyze/collector) on
-your Postgres database server.
-
-The collector sends normalized query data and other statistics to our service
-every 10 minutes.
+In order to use pganalyze you need to install the [pganalyze collector](https://github.com/pganalyze/collector)
+in your environment, either on the database server directly, or a container / VM
+close to it when using managed database services. The collector sends query data
```

Comment:
```suggestion
in your environment, either on the database server directly, or a container / VM
that can connect to it when using managed database services. The collector sends query data
```
maybe? I think "close to it" is vague (I'm guessing "close" in terms of network topology, but it's not clear), and not strictly required: if you really want your collector running in a different region, that's not ideal, but it should work.

Reply:
We could also consider
```suggestion
in your environment, either on the database server directly, or a container / VM
that can connect to it. The collector sends query data
```
since you may want to or need to monitor your self-managed database server with the collector installed elsewhere. That's probably a niche configuration, though, so we could keep the "managed services" part.

Reply:
I'd say that we can keep the managed database services part. I understand your point and it is true, but saying "managed database services" there covers the majority cases and they can feel like "yes that's my case", which is helpful.

RE: changing "close to" to "that can connect to", I was actually thinking the same thing too, so I'm +1 👍 This is copied from the Overview overview page so I'll update there too